packages/zcli-apps/package.json

```diff
@@ -25,6 +25,7 @@
     "express": "^4.17.1",
     "form-data": "^4.0.0",
     "fs-extra": "^10.0.0",
+    "jszip": "^3.10.1",
```

Comment:
Curious about the introduction of `jszip` to the project when we already have a similar package `adm-zip` [here](https://github.com/zendesk/zcli/blob/0770d8cfec583a5f31d517f5517c4a735a1a7f7d/packages/zcli-apps/package.json#L20)

Reply:
Thanks, I didn't noticed. Changed from jszip to adm-zip in recent push